File: sem3 pdfs/dsacodes/week_2/linearsearch.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
	int sear,n,arr[100],i;
	cout<<"enter size of array";
	cin>>n;
	cout<<"enter elemnets";
	for(i=0;i<n;i++)
		cin>>arr[i];
	cout<<"enter searches element";
	cin>>sear;
	for(i=0;i<n;i++)
	{
		if(arr[i]==sear)
		{
			cout<<"Searched element "<<sear<<" is found at index "<<i;
			break;
		}
	}
	if(i==n)
		cout<<"Search element not found";
}

File: sem4 pdfs/DAA lab/WEEK_2/Linearsearch.java
import java.util.Scanner;
class Linearsearch{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter size of array");
        int n=sc.nextInt();
        System.out.println("Enter search element");
        int sear=sc.nextInt();
        int arr[]=new int[n];
        int i,flag;
        System.out.println("Enter array elements");
        for(i=0;i<n;i++){
            arr[i]=sc.nextInt();
        }
        for(i=0;i<n;i++){
            if(sear==arr[i]){
                break;
            }
            if(i==n-1){
                System.out.println("Element not found");
                return ;
            }
        }
        System.out.println("element found at index "+i);
        
        
        
    }
}
